plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1126,6 +1128,19 @@ public boolean isLegacyHiveViewTranslation()
         return this.legacyHiveViewTranslation;
     }
 
+    @Config("hive.run-hive-views-as-invoker")
```

Comment:
This is a 3rd config toggle pertaining to Hive Views translation.
I think  we should group the configs together under one namespace: `hive.hive-views.*`.

I posted a PR to discuss that: https://github.com/trinodb/trino/pull/12238

Reply:
I updated the config name to follow what is being proposed in #12238